ais/api/views.py

```diff
@@ -395,18 +477,14 @@ def search_view(query):
                 'include_units' in request.args,
                 is_range=high_num is not None,
                 is_unit=unit_type is not None) \
-                .exclude_non_opa('opa_only' in request.args)
+                .exclude_non_opa('opa_only' in request.args) \
```

Comment:
another thought - this is hacky but we could add this directly after where passyunk gets called to force the `address` search type.

    if search_type == 'none' and ';' in query:
        search_type = 'address'

then a query like `1234 MARKET ST;1200 WALNUT ST` will get passed on to the addresses endpoint which should handle it in batch mode.